shared/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -153,6 +153,85 @@ namespace datadog::shared::nativeloader
                     return CORPROF_E_PROFILER_CANCEL_ACTIVATION;
                 }
             }
+
+            // If we weren't on the explicit include list, then try to filter out `dotnet build` etc.
+            // We don't want to instrument _build_ processes in dotnet by default, as they generally
+            // don't give useful information, add latency, and risk triggering bugs in the runtime,
+            // particularly around shutdown, like this one: https://github.com/dotnet/runtime/issues/55441
+            const auto command_line = GetCurrentProcessCommandLine();
+            const auto process_command_line = std::get<0>(command_line);
+            const auto command_line_arguments = std::get<1>(command_line);
```

Comment:
oh I see the `command_line_arguments[1]` made me realized that it's not the command-line arguments but the tokenized command-line.

Reply:
ah, yes, you're right, it's not just the arguments, I'll rename some variables 🙂

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/5564/commits/594e711959b590150fd6f1f1aeb2f7c222b239b8

Renamed to `tokenized_command_line` and using `token` instead of `arg` generally